gradle.properties

```diff
@@ -1,2 +1 @@
 version=2.12.0-SNAPSHOT
-snakeyaml.version=2.2
```

Comment:
See https://github.com/spring-projects/spring-boot/commit/791e6dcc29324c60ed90377dbdc999f0a29e1e73 for more.